instrumentation/vertx/vertx-sql-client-4.0/javaagent/build.gradle.kts

```diff
@@ -12,13 +12,39 @@ muzzle {
 }
 
 dependencies {
-  library("io.vertx:vertx-sql-client:4.0.0")
-  compileOnly("io.vertx:vertx-codegen:4.0.0")
+  val version = "4.0.0"
+  library("io.vertx:vertx-sql-client:$version")
+  compileOnly("io.vertx:vertx-codegen:$version")
 
   testInstrumentation(project(":instrumentation:netty:netty-4.1:javaagent"))
 
-  testLibrary("io.vertx:vertx-pg-client:4.0.0")
-  testLibrary("io.vertx:vertx-codegen:4.0.0")
+  testLibrary("io.vertx:vertx-pg-client:$version")
+  testLibrary("io.vertx:vertx-codegen:$version")
+}
+
+testing {
```

Comment:
Why do you need this suite? Try running `./gradlew :instrumentation:vertx:vertx-sql-client-4.0:javaagent:build -PtestLatestDeps=true` doesn't it already run the `testStableSemconv` task with the latest dependency?

Reply:
good point - I had originally thought that the test would need to be different - but it turned out that it doesn't